tests/functional/sync_v3_sign_in.js

```diff
@@ -11,8 +11,9 @@ const selectors = require('./lib/selectors');
 const uaStrings = require('./lib/ua-strings');
 
 const config = intern._config;
-const PAGE_URL = `${config.fxaContentRoot}signin?context=fx_desktop_v3&service=sync&forceAboutAccounts=true&automatedBrowser=true`;
-const TOKEN_CODE_PAGE_URL = `${config.fxaContentRoot}signin?context=fx_desktop_v3&service=sync`;
+const SIGNIN_PAGE_URL = `${config.fxaContentRoot}signin?context=fx_desktop_v3&service=sync&automatedBrowser=true`;
+const ENTER_EMAIL_PAGE_URL = `${config.fxaContentRoot}?context=fx_desktop_v3&service=sync&automatedBrowser=true&forceAboutAccounts=true`;
```

Comment:
nit: I wonder if it makes sense to call out the email first property of this page in the variable name?

Reply:
I'm sure it does.